### ISSUE TITLE
[JENKINS-43106] pipeline job hangs forever at checkout GitSCM - after git rev-list (sporadic)

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jenkins#21970]
<p>Sometimes a pipeline job hangs around checkout. </p>

<p>Lately this seems to be after git rev-list. </p>

<p> </p>

<pre>
 > git rev-list --no-walk db25209b84b12cca5214227a7999fe280c82017e # timeout=10</pre>
<p>It will hang here until job is killed. A restart is fine - and it only happens occasionally. </p>

<p>This has been more commonly reported since 2018 February/January, a recent update. </p>

<p> </p>

<p>A support bundle of this when it was happening: <span class="nobr"><a href="https://issues.jenkins.io/secure/attachment/41376/41376_support_2018-02-07_22.54.25.zip" title="support_2018-02-07_22.54.25.zip attached to JENKINS-43106">support_2018-02-07_22.54.25.zip<sup><img class="rendericon" src="https://issues.jenkins.io/images/icons/link_attachment_7.gif" height="7" width="7" align="absmiddle" alt="" border="0"/></sup></a></span></p>

<p>IT was happening on slave-05 at the time (can see in its thread dump a few waiting things). Timeout does not work, it will wait indefinitely. </p>

<p> </p>

<p> </p>
<hr />
<p>My pipeline job is getting sporadically hung on checkout step.</p>

<p>Our git repo is relatively small and clone should not take more than 1 min.</p>

<p>The problem occurring really often once a couple builds. Once it's occurring, the only way that I know to get rid of this state so far is, is disconnect the slave and re-connect it back.</p>

<p>Build log:</p>

<p>Started by user <USER><br/>
 Checking out git <REPO> to read Jenkinsfile.bootstrap<br/>
 > git rev-parse --is-inside-work-tree # timeout=10<br/>
 Fetching changes from the remote Git repository<br/>
 > git config remote.origin.url <REPO> # timeout=10<br/>
 Fetching upstream changes from <REPO><br/>
 > git --version # timeout=10<br/>
 using GIT_SSH to set credentials jenkinsslave SSH key<br/>
 > git fetch --tags --progress <REPO> +refs/heads/<b>:refs/remotes/origin/</b><br/>
 > git rev-parse refs/remotes/origin/devops_jenkins^{commit} # timeout=10<br/>
 > git rev-parse refs/remotes/origin/origin/devops_jenkins^{commit} # timeout=10<br/>
 Checking out Revision 2fd998fbe8b23f814b57d8c6eb7b46004268eedc (refs/remotes/origin/devops_jenkins)<br/>
 > git config core.sparsecheckout # timeout=10<br/>
 > git checkout -f 2fd998fbe8b23f814b57d8c6eb7b46004268eedc<br/>
 > git rev-list 2fd998fbe8b23f814b57d8c6eb7b46004268eedc # timeout=10<br/>
 <span class="error">&#91;Pipeline&#93;</span> timestamps<br/>
 <span class="error">&#91;Pipeline&#93;</span> {<br/>
 <span class="error">&#91;Pipeline&#93;</span> node<br/>
 00:00:01.249 Running on <SLAVE> in <WORKSPACE><br/>
 <span class="error">&#91;Pipeline&#93;</span> {<br/>
 <span class="error">&#91;Pipeline&#93;</span> stage<br/>
 <span class="error">&#91;Pipeline&#93;</span> { (Load Jenkinsfile)<br/>
 <span class="error">&#91;Pipeline&#93;</span> echo<br/>
 00:00:01.267 Branch: origin/devops_jenkins Repo: <REPO><br/>
 <span class="error">&#91;Pipeline&#93;</span> checkout<br/>
 00:03:27.565 > git rev-parse -<del>is-inside-work-tree # timeout=10 <</del>- This and further lines are being printed only after slave disconnect.<br/>
 00:03:27.566 Fetching changes from the remote Git repository<br/>
 <span class="error">&#91;Pipeline&#93;</span> }<br/>
 <span class="error">&#91;Pipeline&#93;</span> // stage<br/>
 <span class="error">&#91;Pipeline&#93;</span> }<br/>
 <span class="error">&#91;Pipeline&#93;</span> // node<br/>
 <span class="error">&#91;Pipeline&#93;</span> }<br/>
 <span class="error">&#91;Pipeline&#93;</span> // timestamps<br/>
 <span class="error">&#91;Pipeline&#93;</span> End of Pipeline<br/>
 hudson.remoting.RemotingSystemException: java.io.IOException: Backing channel 'jenkinsslave_192.168.45.26' is disconnected.<br/>
 at hudson.remoting.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:272)<br/>
 at com.sun.proxy.$Proxy98.setRemoteUrl(Unknown Source)<br/>
 at org.jenkinsci.plugins.gitclient.RemoteGitImpl.setRemoteUrl(RemoteGitImpl.java:295)<br/>
 at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:794)<br/>
 at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1070)<br/>
 at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1101)<br/>
 at org.jenkinsci.plugins.workflow.steps.scm.SCMStep.checkout(SCMStep.java:109)<br/>
 at org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:83)<br/>
 at org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:73)<br/>
 at org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1$1.call(AbstractSynchronousNonBlockingStepExecution.java:47)<br/>
 at hudson.security.ACL.impersonate(ACL.java:221)<br/>
 at org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1.run(AbstractSynchronousNonBlockingStepExecution.java:44)<br/>
 at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)<br/>
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)<br/>
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)<br/>
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)<br/>
 at java.lang.Thread.run(Thread.java:745)<br/>
 Caused by: java.io.IOException: Backing channel 'jenkinsslave_192.168.45.26' is disconnected.<br/>
 at hudson.remoting.RemoteInvocationHandler.channelOrFail(RemoteInvocationHandler.java:191)<br/>
 at hudson.remoting.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:256)<br/>
 ... 16 more<br/>
 Caused by: hudson.remoting.Channel$OrderlyShutdown<br/>
 at hudson.remoting.Channel$CloseCommand.execute(Channel.java:1121)<br/>
 at hudson.remoting.Channel$1.handle(Channel.java:526)<br/>
 at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:83)<br/>
 Caused by: Command close created at<br/>
 at hudson.remoting.Command.<init>(Command.java:59)<br/>
 at hudson.remoting.Channel$CloseCommand.<init>(Channel.java:1115)<br/>
 at hudson.remoting.Channel$CloseCommand.<init>(Channel.java:1113)<br/>
 at hudson.remoting.Channel.close(Channel.java:1273)<br/>
 at hudson.remoting.Channel.close(Channel.java:1255)<br/>
 at hudson.remoting.Channel$CloseCommand.execute(Channel.java:1120)<br/>
 ... 2 more<br/>
 Finished: FAILURE</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=levt">levt</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-43106" target="_blank">pipeline job hangs forever at checkout GitSCM - after git rev-list (sporadic)</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/vrobert78.jpg" title="vrobert78's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=vrobert78">vrobert78</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Blocker
<li><b>component(s)</b>: core, git-plugin
<li><b>resolution</b>: Duplicate
<li><b>resolved</b>: 2018-02-12T13:46:15+00:00
<li><b>votes</b>: 8
<li><b>watchers</b>: 19
<li><b>imported</b>: 2025-11-24
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Sometimes a pipeline job hangs around checkout. </p>

<p>Lately this seems to be after git rev-list. </p>

<p> </p>
<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java"> &gt; git rev-list --no-walk db25209b84b12cca5214227a7999fe280c82017e # timeout=10</pre>
</div></div>
<p>It will hang here until job is killed. A restart is fine - and it only happens occasionally. </p>

<p>This has been more commonly reported since 2018 February/January, a recent update. </p>

<p> </p>

<p>A support bundle of this when it was happening: <span class="nobr"><a href="https://issues.jenkins.io/secure/attachment/41376/41376_support_2018-02-07_22.54.25.zip" title="support_2018-02-07_22.54.25.zip attached to JENKINS-43106">support_2018-02-07_22.54.25.zip<sup><img class="rendericon" src="https://issues.jenkins.io/images/icons/link_attachment_7.gif" height="7" width="7" align="absmiddle" alt="" border="0"/></sup></a></span></p>

<p>IT was happening on slave-05 at the time (can see in its thread dump a few waiting things). Timeout does not work, it will wait indefinitely. </p>

<p> </p>

<p> </p>
<hr />
<p>My pipeline job is getting sporadically hung on checkout step.</p>

<p>Our git repo is relatively small and clone should not take more than 1 min.</p>

<p>The problem occurring really often once a couple builds. Once it's occurring, the only way that I know to get rid of this state so far is, is disconnect the slave and re-connect it back.</p>

<p>Build log:</p>

<p>Started by user &lt;USER&gt;
 Checking out git &lt;REPO&gt; to read Jenkinsfile.bootstrap
 &gt; git rev-parse --is-inside-work-tree # timeout=10
 Fetching changes from the remote Git repository
 &gt; git config remote.origin.url &lt;REPO&gt; # timeout=10
 Fetching upstream changes from &lt;REPO&gt;
 &gt; git --version # timeout=10
 using GIT_SSH to set credentials jenkinsslave SSH key
 &gt; git fetch --tags --progress &lt;REPO&gt; +refs/heads/<b>:refs/remotes/origin/</b>
 &gt; git rev-parse refs/remotes/origin/devops_jenkins^{commit} # timeout=10
 &gt; git rev-parse refs/remotes/origin/origin/devops_jenkins^{commit} # timeout=10
 Checking out Revision 2fd998fbe8b23f814b57d8c6eb7b46004268eedc (refs/remotes/origin/devops_jenkins)
 &gt; git config core.sparsecheckout # timeout=10
 &gt; git checkout -f 2fd998fbe8b23f814b57d8c6eb7b46004268eedc
 &gt; git rev-list 2fd998fbe8b23f814b57d8c6eb7b46004268eedc # timeout=10
 <span class="error">&#91;Pipeline&#93;</span> timestamps
 <span class="error">&#91;Pipeline&#93;</span> {
 <span class="error">&#91;Pipeline&#93;</span> node
 00:00:01.249 Running on &lt;SLAVE&gt; in &lt;WORKSPACE&gt;
 <span class="error">&#91;Pipeline&#93;</span> {
 <span class="error">&#91;Pipeline&#93;</span> stage
 <span class="error">&#91;Pipeline&#93;</span> { (Load Jenkinsfile)
 <span class="error">&#91;Pipeline&#93;</span> echo
 00:00:01.267 Branch: origin/devops_jenkins Repo: &lt;REPO&gt;
 <span class="error">&#91;Pipeline&#93;</span> checkout
 00:03:27.565 &gt; git rev-parse -<del>is-inside-work-tree # timeout=10 &lt;</del>- This and further lines are being printed only after slave disconnect.
 00:03:27.566 Fetching changes from the remote Git repository
 <span class="error">&#91;Pipeline&#93;</span> }
 <span class="error">&#91;Pipeline&#93;</span> // stage
 <span class="error">&#91;Pipeline&#93;</span> }
 <span class="error">&#91;Pipeline&#93;</span> // node
 <span class="error">&#91;Pipeline&#93;</span> }
 <span class="error">&#91;Pipeline&#93;</span> // timestamps
 <span class="error">&#91;Pipeline&#93;</span> End of Pipeline
 hudson.remoting.RemotingSystemException: java.io.IOException: Backing channel 'jenkinsslave_192.168.45.26' is disconnected.
 at hudson.remoting.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:272)
 at com.sun.proxy.$Proxy98.setRemoteUrl(Unknown Source)
 at org.jenkinsci.plugins.gitclient.RemoteGitImpl.setRemoteUrl(RemoteGitImpl.java:295)
 at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:794)
 at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1070)
 at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1101)
 at org.jenkinsci.plugins.workflow.steps.scm.SCMStep.checkout(SCMStep.java:109)
 at org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:83)
 at org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:73)
 at org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1$1.call(AbstractSynchronousNonBlockingStepExecution.java:47)
 at hudson.security.ACL.impersonate(ACL.java:221)
 at org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1.run(AbstractSynchronousNonBlockingStepExecution.java:44)
 at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
 at java.lang.Thread.run(Thread.java:745)
 Caused by: java.io.IOException: Backing channel 'jenkinsslave_192.168.45.26' is disconnected.
 at hudson.remoting.RemoteInvocationHandler.channelOrFail(RemoteInvocationHandler.java:191)
 at hudson.remoting.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:256)
 ... 16 more
 Caused by: hudson.remoting.Channel$OrderlyShutdown
 at hudson.remoting.Channel$CloseCommand.execute(Channel.java:1121)
 at hudson.remoting.Channel$1.handle(Channel.java:526)
 at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:83)
 Caused by: Command close created at
 at hudson.remoting.Command.&lt;init&gt;(Command.java:59)
 at hudson.remoting.Channel$CloseCommand.&lt;init&gt;(Channel.java:1115)
 at hudson.remoting.Channel$CloseCommand.&lt;init&gt;(Channel.java:1113)
 at hudson.remoting.Channel.close(Channel.java:1273)
 at hudson.remoting.Channel.close(Channel.java:1255)
 at hudson.remoting.Channel$CloseCommand.execute(Channel.java:1120)
 ... 2 more
 Finished: FAILURE</p></pre>
</details>
</details>
<details><summary><i>environment</i></summary>

```
OS: Ubunty 16.04<br/>
Java: openjdk version &quot;1.8.0_121&quot;<br/>
git: 2.7.4<br/>
Jenkins: 2.32.3 (Master and slave running at the same server)<br/>
Plugins:<br/>
Git Client plugin: 2.4.0<br/>
Git plugin: 3.1.0<br/>
Pipeline: 2.5<br/>
Pipeline SCM Step: 2.4<br/>
SCM API Plugin: 2.1.1
```
</details>
<details><summary><i>8 attachments</i></summary>

- [Capture.PNG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/36779/Capture.PNG)
  > ![Capture.PNG](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/36779/Capture.PNG)
- [Jenkinsfile](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/36770/Jenkinsfile)
- [config.xml](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/39431/config.xml)
- [config.xml](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/36769/config.xml)
- [support_2017-10-06_04.20.32.zip](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/39948/support_2017-10-06_04.20.32.zip)
- [support_2017-10-11_21.17.12.zip](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/40003/support_2017-10-11_21.17.12.zip)
- [support_2018-02-07_22.54.25.zip](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/41376/support_2018-02-07_22.54.25.zip)
- [threadDump.txt](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/36771/threadDump.txt)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>tapsa:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-43106?focusedId=315487&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-315487">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>This happen also in our CI environment:</p>

<p><a href="https://jenkins.io/" class="external-link" target="_blank" rel="nofollow noopener">Jenkins ver. 2.53</a></p>

<p>and plugins are:</p>

<p>&lt;plugins&gt;&lt;shortName&gt;cloudbees-folder&lt;/shortName&gt;&lt;version&gt;6.1.2&lt;/version&gt;&lt;shortName&gt;junit&lt;/shortName&gt;&lt;version&gt;1.21&lt;/version&gt;&lt;shortName&gt;ssh-credentials&lt;/shortName&gt;&lt;version&gt;1.13&lt;/version&gt;&lt;shortName&gt;script-security&lt;/shortName&gt;&lt;version&gt;1.34&lt;/version&gt;&lt;shortName&gt;matrix-project&lt;/shortName&gt;&lt;version&gt;1.11&lt;/version&gt;&lt;shortName&gt;antisamy-markup-formatter&lt;/shortName&gt;&lt;version&gt;1.5&lt;/version&gt;&lt;shortName&gt;github&lt;/shortName&gt;&lt;version&gt;1.28.0&lt;/version&gt;&lt;shortName&gt;pam-auth&lt;/shortName&gt;&lt;version&gt;1.3&lt;/version&gt;&lt;shortName&gt;git-client&lt;/shortName&gt;&lt;version&gt;2.5.0&lt;/version&gt;&lt;shortName&gt;windows-slaves&lt;/shortName&gt;&lt;version&gt;1.3.1&lt;/version&gt;&lt;shortName&gt;mailer&lt;/shortName&gt;&lt;version&gt;1.20&lt;/version&gt;&lt;shortName&gt;ldap&lt;/shortName&gt;&lt;version&gt;1.17&lt;/version&gt;&lt;shortName&gt;token-macro&lt;/shortName&gt;&lt;version&gt;2.3&lt;/version&gt;&lt;shortName&gt;mapdb-api&lt;/shortName&gt;&lt;version&gt;1.0.9.0&lt;/version&gt;&lt;shortName&gt;icon-shim&lt;/shortName&gt;&lt;version&gt;2.0.3&lt;/version&gt;&lt;shortName&gt;matrix-auth&lt;/shortName&gt;&lt;version&gt;1.5&lt;/version&gt;&lt;shortName&gt;pipeline-stage-step&lt;/shortName&gt;&lt;version&gt;2.2&lt;/version&gt;&lt;shortName&gt;external-monitor-job&lt;/shortName&gt;&lt;version&gt;1.7&lt;/version&gt;&lt;shortName&gt;subversion&lt;/shortName&gt;&lt;version&gt;2.9&lt;/version&gt;&lt;shortName&gt;build-timeout&lt;/shortName&gt;&lt;version&gt;1.18&lt;/version&gt;&lt;shortName&gt;credentials&lt;/shortName&gt;&lt;version&gt;2.1.15&lt;/version&gt;&lt;shortName&gt;ssh-slaves&lt;/shortName&gt;&lt;version&gt;1.15&lt;/version&gt;&lt;shortName&gt;structs&lt;/shortName&gt;&lt;version&gt;1.10&lt;/version&gt;&lt;shortName&gt;workflow-step-api&lt;/shortName&gt;&lt;version&gt;2.13&lt;/version&gt;&lt;shortName&gt;workflow-basic-steps&lt;/shortName&gt;&lt;version&gt;2.6&lt;/version&gt;&lt;shortName&gt;plain-credentials&lt;/shortName&gt;&lt;version&gt;1.4&lt;/version&gt;&lt;shortName&gt;credentials-binding&lt;/shortName&gt;&lt;version&gt;1.13&lt;/version&gt;&lt;shortName&gt;timestamper&lt;/shortName&gt;&lt;version&gt;1.8.8&lt;/version&gt;&lt;shortName&gt;email-ext&lt;/shortName&gt;&lt;version&gt;2.59&lt;/version&gt;&lt;shortName&gt;ws-cleanup&lt;/shortName&gt;&lt;version&gt;0.34&lt;/version&gt;&lt;shortName&gt;jira&lt;/shortName&gt;&lt;version&gt;2.3&lt;/version&gt;&lt;shortName&gt;ant&lt;/shortName&gt;&lt;version&gt;1.7&lt;/version&gt;&lt;shortName&gt;gradle&lt;/shortName&gt;&lt;version&gt;1.27.1&lt;/version&gt;&lt;shortName&gt;workflow-aggregator&lt;/shortName&gt;&lt;version&gt;2.5&lt;/version&gt;&lt;shortName&gt;jquery-detached&lt;/shortName&gt;&lt;version&gt;1.2.1&lt;/version&gt;&lt;shortName&gt;crowd2&lt;/shortName&gt;&lt;version&gt;1.8&lt;/version&gt;&lt;shortName&gt;durable-task&lt;/shortName&gt;&lt;version&gt;1.14&lt;/version&gt;&lt;shortName&gt;workflow-api&lt;/shortName&gt;&lt;version&gt;2.20&lt;/version&gt;&lt;shortName&gt;workflow-support&lt;/shortName&gt;&lt;version&gt;2.14&lt;/version&gt;&lt;shortName&gt;monitoring&lt;/shortName&gt;&lt;version&gt;1.69.1&lt;/version&gt;&lt;shortName&gt;workflow-job&lt;/shortName&gt;&lt;version&gt;2.10&lt;/version&gt;&lt;shortName&gt;javadoc&lt;/shortName&gt;&lt;version&gt;1.4&lt;/version&gt;&lt;shortName&gt;pipeline-rest-api&lt;/shortName&gt;&lt;version&gt;2.9&lt;/version&gt;&lt;shortName&gt;handlebars&lt;/shortName&gt;&lt;version&gt;1.1.1&lt;/version&gt;&lt;shortName&gt;maven-plugin&lt;/shortName&gt;&lt;version&gt;2.15.1&lt;/version&gt;&lt;shortName&gt;momentjs&lt;/shortName&gt;&lt;version&gt;1.1.1&lt;/version&gt;&lt;shortName&gt;pipeline-stage-view&lt;/shortName&gt;&lt;version&gt;2.9&lt;/version&gt;&lt;shortName&gt;github-api&lt;/shortName&gt;&lt;version&gt;1.86&lt;/version&gt;&lt;shortName&gt;pipeline-build-step&lt;/shortName&gt;&lt;version&gt;2.5.1&lt;/version&gt;&lt;shortName&gt;jobConfigHistory&lt;/shortName&gt;&lt;version&gt;2.18&lt;/version&gt;&lt;shortName&gt;ace-editor&lt;/shortName&gt;&lt;version&gt;1.1&lt;/version&gt;&lt;shortName&gt;ssh-agent&lt;/shortName&gt;&lt;version&gt;1.15&lt;/version&gt;&lt;shortName&gt;workflow-scm-step&lt;/shortName&gt;&lt;version&gt;2.4&lt;/version&gt;&lt;shortName&gt;scm-api&lt;/shortName&gt;&lt;version&gt;2.2.2&lt;/version&gt;&lt;shortName&gt;workflow-cps&lt;/shortName&gt;&lt;version&gt;2.40&lt;/version&gt;&lt;shortName&gt;git-server&lt;/shortName&gt;&lt;version&gt;1.7&lt;/version&gt;&lt;shortName&gt;workflow-cps-global-lib&lt;/shortName&gt;&lt;version&gt;2.9&lt;/version&gt;&lt;shortName&gt;thinBackup&lt;/shortName&gt;&lt;version&gt;1.9&lt;/version&gt;&lt;shortName&gt;branch-api&lt;/shortName&gt;&lt;version&gt;2.0.11&lt;/version&gt;&lt;shortName&gt;git&lt;/shortName&gt;&lt;version&gt;3.5.1&lt;/version&gt;&lt;shortName&gt;workflow-multibranch&lt;/shortName&gt;&lt;version&gt;2.16&lt;/version&gt;&lt;shortName&gt;workflow-durable-task-step&lt;/shortName&gt;&lt;version&gt;2.15&lt;/version&gt;&lt;shortName&gt;pipeline-input-step&lt;/shortName&gt;&lt;version&gt;2.8&lt;/version&gt;&lt;shortName&gt;urltrigger&lt;/shortName&gt;&lt;version&gt;0.41&lt;/version&gt;&lt;shortName&gt;shelve-project-plugin&lt;/shortName&gt;&lt;version&gt;1.5&lt;/version&gt;&lt;shortName&gt;jquery&lt;/shortName&gt;&lt;version&gt;1.11.2-1&lt;/version&gt;&lt;shortName&gt;analysis-core&lt;/shortName&gt;&lt;version&gt;1.92&lt;/version&gt;&lt;shortName&gt;audit-trail&lt;/shortName&gt;&lt;version&gt;2.2&lt;/version&gt;&lt;shortName&gt;StashBranchParameter&lt;/shortName&gt;&lt;version&gt;0.3.0&lt;/version&gt;&lt;shortName&gt;job-import-plugin&lt;/shortName&gt;&lt;version&gt;1.5&lt;/version&gt;&lt;shortName&gt;parameterized-trigger&lt;/shortName&gt;&lt;version&gt;2.35.2&lt;/version&gt;&lt;shortName&gt;robot&lt;/shortName&gt;&lt;version&gt;1.6.4&lt;/version&gt;&lt;shortName&gt;rebuild&lt;/shortName&gt;&lt;version&gt;1.25&lt;/version&gt;&lt;shortName&gt;nested-view&lt;/shortName&gt;&lt;version&gt;1.14&lt;/version&gt;&lt;shortName&gt;claim&lt;/shortName&gt;&lt;version&gt;2.9&lt;/version&gt;&lt;shortName&gt;stashNotifier&lt;/shortName&gt;&lt;version&gt;1.11.6&lt;/version&gt;&lt;shortName&gt;envinject&lt;/shortName&gt;&lt;version&gt;1.93.1&lt;/version&gt;&lt;shortName&gt;mask-passwords&lt;/shortName&gt;&lt;version&gt;2.10.1&lt;/version&gt;&lt;shortName&gt;github-branch-source&lt;/shortName&gt;&lt;version&gt;2.0.5&lt;/version&gt;&lt;shortName&gt;radiatorviewplugin&lt;/shortName&gt;&lt;version&gt;1.29&lt;/version&gt;&lt;shortName&gt;display-url-api&lt;/shortName&gt;&lt;version&gt;2.0&lt;/version&gt;&lt;shortName&gt;analysis-collector&lt;/shortName&gt;&lt;version&gt;1.52&lt;/version&gt;&lt;shortName&gt;publish-over-ssh&lt;/shortName&gt;&lt;version&gt;1.17&lt;/version&gt;&lt;shortName&gt;xunit&lt;/shortName&gt;&lt;version&gt;1.102&lt;/version&gt;&lt;shortName&gt;job-node-stalker&lt;/shortName&gt;&lt;version&gt;1.0.5&lt;/version&gt;&lt;shortName&gt;run-condition&lt;/shortName&gt;&lt;version&gt;1.0&lt;/version&gt;&lt;shortName&gt;scp&lt;/shortName&gt;&lt;version&gt;1.8&lt;/version&gt;&lt;shortName&gt;msbuild&lt;/shortName&gt;&lt;version&gt;1.27&lt;/version&gt;&lt;shortName&gt;bouncycastle-api&lt;/shortName&gt;&lt;version&gt;2.16.2&lt;/version&gt;&lt;shortName&gt;git-parameter&lt;/shortName&gt;&lt;version&gt;0.8.1&lt;/version&gt;&lt;shortName&gt;cppcheck&lt;/shortName&gt;&lt;version&gt;1.21&lt;/version&gt;&lt;shortName&gt;config-file-provider&lt;/shortName&gt;&lt;version&gt;2.16.4&lt;/version&gt;&lt;shortName&gt;copyartifact&lt;/shortName&gt;&lt;version&gt;1.38.1&lt;/version&gt;&lt;shortName&gt;conditional-buildstep&lt;/shortName&gt;&lt;version&gt;1.3.6&lt;/version&gt;&lt;shortName&gt;jacoco&lt;/shortName&gt;&lt;version&gt;2.2.1&lt;/version&gt;&lt;shortName&gt;github-organization-folder&lt;/shortName&gt;&lt;version&gt;1.6&lt;/version&gt;&lt;shortName&gt;ivy&lt;/shortName&gt;&lt;version&gt;1.27.1&lt;/version&gt;&lt;shortName&gt;emma&lt;/shortName&gt;&lt;version&gt;1.29&lt;/version&gt;&lt;shortName&gt;build-user-vars-plugin&lt;/shortName&gt;&lt;version&gt;1.5&lt;/version&gt;&lt;shortName&gt;sonar&lt;/shortName&gt;&lt;version&gt;2.6.1&lt;/version&gt;&lt;shortName&gt;warnings&lt;/shortName&gt;&lt;version&gt;4.63&lt;/version&gt;&lt;shortName&gt;xvfb&lt;/shortName&gt;&lt;version&gt;1.1.3&lt;/version&gt;&lt;shortName&gt;embeddable-build-status&lt;/shortName&gt;&lt;version&gt;1.9&lt;/version&gt;&lt;shortName&gt;text-finder&lt;/shortName&gt;&lt;version&gt;1.10&lt;/version&gt;&lt;shortName&gt;pipeline-stage-tags-metadata&lt;/shortName&gt;&lt;version&gt;1.1.9&lt;/version&gt;&lt;shortName&gt;docker-commons&lt;/shortName&gt;&lt;version&gt;1.8&lt;/version&gt;&lt;shortName&gt;resource-disposer&lt;/shortName&gt;&lt;version&gt;0.7&lt;/version&gt;&lt;shortName&gt;docker-workflow&lt;/shortName&gt;&lt;version&gt;1.13&lt;/version&gt;&lt;shortName&gt;jenkins-flowdock-plugin&lt;/shortName&gt;&lt;version&gt;1.1.8&lt;/version&gt;&lt;shortName&gt;jackson2-api&lt;/shortName&gt;&lt;version&gt;2.7.3&lt;/version&gt;&lt;shortName&gt;pipeline-github-lib&lt;/shortName&gt;&lt;version&gt;1.0&lt;/version&gt;&lt;shortName&gt;stash-pullrequest-builder&lt;/shortName&gt;&lt;version&gt;1.7.0&lt;/version&gt;&lt;shortName&gt;pipeline-model-declarative-agent&lt;/shortName&gt;&lt;version&gt;1.1.1&lt;/version&gt;&lt;shortName&gt;pipeline-model-extensions&lt;/shortName&gt;&lt;version&gt;1.1.4&lt;/version&gt;&lt;shortName&gt;pipeline-graph-analysis&lt;/shortName&gt;&lt;version&gt;1.5&lt;/version&gt;&lt;shortName&gt;checkstyle&lt;/shortName&gt;&lt;version&gt;3.49&lt;/version&gt;&lt;shortName&gt;pipeline-milestone-step&lt;/shortName&gt;&lt;version&gt;1.3.1&lt;/version&gt;&lt;shortName&gt;pipeline-model-api&lt;/shortName&gt;&lt;version&gt;1.1.9&lt;/version&gt;&lt;shortName&gt;pipeline-model-definition&lt;/shortName&gt;&lt;version&gt;1.1.4&lt;/version&gt;&lt;shortName&gt;promoted-builds&lt;/shortName&gt;&lt;version&gt;2.29.1&lt;/version&gt;&lt;shortName&gt;build-name-setter&lt;/shortName&gt;&lt;version&gt;1.6.7&lt;/version&gt;&lt;shortName&gt;multiple-scms&lt;/shortName&gt;&lt;version&gt;0.6&lt;/version&gt;&lt;shortName&gt;valgrind&lt;/shortName&gt;&lt;version&gt;0.27&lt;/version&gt;&lt;shortName&gt;build-monitor-plugin&lt;/shortName&gt;&lt;version&gt;1.12+build.201704111018&lt;/version&gt;&lt;shortName&gt;authentication-tokens&lt;/shortName&gt;&lt;version&gt;1.3&lt;/version&gt;&lt;shortName&gt;sitemonitor&lt;/shortName&gt;&lt;version&gt;0.5&lt;/version&gt;&lt;shortName&gt;next-build-number&lt;/shortName&gt;&lt;version&gt;1.4&lt;/version&gt;&lt;shortName&gt;htmlpublisher&lt;/shortName&gt;&lt;version&gt;1.14&lt;/version&gt;&lt;shortName&gt;description-setter&lt;/shortName&gt;&lt;version&gt;1.10&lt;/version&gt;&lt;/plugins&gt;</p>
</pre>
</details>

</ul>
</details>

<p>This happen also in our CI environment:</p>

<p><a href="https://jenkins.io/" class="external-link" target="_blank" rel="nofollow noopener">Jenkins ver. 2.53</a></p>

<p>and plugins are:</p>

<p><plugins><shortName>cloudbees-folder</shortName><version>6.1.2</version><shortName>junit</shortName><version>1.21</version><shortName>ssh-credentials</shortName><version>1.13</version><shortName>script-security</shortName><version>1.34</version><shortName>matrix-project</shortName><version>1.11</version><shortName>antisamy-markup-formatter</shortName><version>1.5</version><shortName>github</shortName><version>1.28.0</version><shortName>pam-auth</shortName><version>1.3</version><shortName>git-client</shortName><version>2.5.0</version><shortName>windows-slaves</shortName><version>1.3.1</version><shortName>mailer</shortName><version>1.20</version><shortName>ldap</shortName><version>1.17</version><shortName>token-macro</shortName><version>2.3</version><shortName>mapdb-api</shortName><version>1.0.9.0</version><shortName>icon-shim</shortName><version>2.0.3</version><shortName>matrix-auth</shortName><version>1.5</version><shortName>pipeline-stage-step</shortName><version>2.2</version><shortName>external-monitor-job</shortName><version>1.7</version><shortName>subversion</shortName><version>2.9</version><shortName>build-timeout</shortName><version>1.18</version><shortName>credentials</shortName><version>2.1.15</version><shortName>ssh-slaves</shortName><version>1.15</version><shortName>structs</shortName><version>1.10</version><shortName>workflow-step-api</shortName><version>2.13</version><shortName>workflow-basic-steps</shortName><version>2.6</version><shortName>plain-credentials</shortName><version>1.4</version><shortName>credentials-binding</shortName><version>1.13</version><shortName>timestamper</shortName><version>1.8.8</version><shortName>email-ext</shortName><version>2.59</version><shortName>ws-cleanup</shortName><version>0.34</version><shortName>jira</shortName><version>2.3</version><shortName>ant</shortName><version>1.7</version><shortName>gradle</shortName><version>1.27.1</version><shortName>workflow-aggregator</shortName><version>2.5</version><shortName>jquery-detached</shortName><version>1.2.1</version><shortName>crowd2</shortName><version>1.8</version><shortName>durable-task</shortName><version>1.14</version><shortName>workflow-api</shortName><version>2.20</version><shortName>workflow-support</shortName><version>2.14</version><shortName>monitoring</shortName><version>1.69.1</version><shortName>workflow-job</shortName><version>2.10</version><shortName>javadoc</shortName><version>1.4</version><shortName>pipeline-rest-api</shortName><version>2.9</version><shortName>handlebars</shortName><version>1.1.1</version><shortName>maven-plugin</shortName><version>2.15.1</version><shortName>momentjs</shortName><version>1.1.1</version><shortName>pipeline-stage-view</shortName><version>2.9</version><shortName>github-api</shortName><version>1.86</version><shortName>pipeline-build-step</shortName><version>2.5.1</version><shortName>jobConfigHistory</shortName><version>2.18</version><shortName>ace-editor</shortName><version>1.1</version><shortName>ssh-agent</shortName><version>1.15</version><shortName>workflow-scm-step</shortName><version>2.4</version><shortName>scm-api</shortName><version>2.2.2</version><shortName>workflow-cps</shortName><version>2.40</version><shortName>git-server</shortName><version>1.7</version><shortName>workflow-cps-global-lib</shortName><version>2.9</version><shortName>thinBackup</shortName><version>1.9</version><shortName>branch-api</shortName><version>2.0.11</version><shortName>git</shortName><version>3.5.1</version><shortName>workflow-multibranch</shortName><version>2.16</version><shortName>workflow-durable-task-step</shortName><version>2.15</version><shortName>pipeline-input-step</shortName><version>2.8</version><shortName>urltrigger</shortName><version>0.41</version><shortName>shelve-project-plugin</shortName><version>1.5</version><shortName>jquery</shortName><version>1.11.2-1</version><shortName>analysis-core</shortName><version>1.92</version><shortName>audit-trail</shortName><version>2.2</version><shortName>StashBranchParameter</shortName><version>0.3.0</version><shortName>job-import-plugin</shortName><version>1.5</version><shortName>parameterized-trigger</shortName><version>2.35.2</version><shortName>robot</shortName><version>1.6.4</version><shortName>rebuild</shortName><version>1.25</version><shortName>nested-view</shortName><version>1.14</version><shortName>claim</shortName><version>2.9</version><shortName>stashNotifier</shortName><version>1.11.6</version><shortName>envinject</shortName><version>1.93.1</version><shortName>mask-passwords</shortName><version>2.10.1</version><shortName>github-branch-source</shortName><version>2.0.5</version><shortName>radiatorviewplugin</shortName><version>1.29</version><shortName>display-url-api</shortName><version>2.0</version><shortName>analysis-collector</shortName><version>1.52</version><shortName>publish-over-ssh</shortName><version>1.17</version><shortName>xunit</shortName><version>1.102</version><shortName>job-node-stalker</shortName><version>1.0.5</version><shortName>run-condition</shortName><version>1.0</version><shortName>scp</shortName><version>1.8</version><shortName>msbuild</shortName><version>1.27</version><shortName>bouncycastle-api</shortName><version>2.16.2</version><shortName>git-parameter</shortName><version>0.8.1</version><shortName>cppcheck</shortName><version>1.21</version><shortName>config-file-provider</shortName><version>2.16.4</version><shortName>copyartifact</shortName><version>1.38.1</version><shortName>conditional-buildstep</shortName><version>1.3.6</version><shortName>jacoco</shortName><version>2.2.1</version><shortName>github-organization-folder</shortName><version>1.6</version><shortName>ivy</shortName><version>1.27.1</version><shortName>emma</shortName><version>1.29</version><shortName>build-user-vars-plugin</shortName><version>1.5</version><shortName>sonar</shortName><version>2.6.1</version><shortName>warnings</shortName><version>4.63</version><shortName>xvfb</shortName><version>1.1.3</version><shortName>embeddable-build-status</shortName><version>1.9</version><shortName>text-finder</shortName><version>1.10</version><shortName>pipeline-stage-tags-metadata</shortName><version>1.1.9</version><shortName>docker-commons</shortName><version>1.8</version><shortName>resource-disposer</shortName><version>0.7</version><shortName>docker-workflow</shortName><version>1.13</version><shortName>jenkins-flowdock-plugin</shortName><version>1.1.8</version><shortName>jackson2-api</shortName><version>2.7.3</version><shortName>pipeline-github-lib</shortName><version>1.0</version><shortName>stash-pullrequest-builder</shortName><version>1.7.0</version><shortName>pipeline-model-declarative-agent</shortName><version>1.1.1</version><shortName>pipeline-model-extensions</shortName><version>1.1.4</version><shortName>pipeline-graph-analysis</shortName><version>1.5</version><shortName>checkstyle</shortName><version>3.49</version><shortName>pipeline-milestone-step</shortName><version>1.3.1</version><shortName>pipeline-model-api</shortName><version>1.1.9</version><shortName>pipeline-model-definition</shortName><version>1.1.4</version><shortName>promoted-builds</shortName><version>2.29.1</version><shortName>build-name-setter</shortName><version>1.6.7</version><shortName>multiple-scms</shortName><version>0.6</version><shortName>valgrind</shortName><version>0.27</version><shortName>build-monitor-plugin</shortName><version>1.12+build.201704111018</version><shortName>authentication-tokens</shortName><version>1.3</version><shortName>sitemonitor</shortName><version>0.5</version><shortName>next-build-number</shortName><version>1.4</version><shortName>htmlpublisher</shortName><version>1.14</version><shortName>description-setter</shortName><version>1.10</version></plugins></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jamesdumay.png" title="jamesdumay's avatar" /> jamesdumay:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-43106?focusedId=316015&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-316015">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hi there,</p>

<p>Could you please generate and upload a support bundle using the <a href="https://wiki.jenkins.io/display/JENKINS/Support+Core+Plugin" class="external-link" target="_blank" rel="nofollow noopener">support core</a> plugin to provide further information here?</p>

<p>Thanks,
James</p>
</pre>
</details>

</ul>
</details>

<p>Hi there,</p>

<p>Could you please generate and upload a support bundle using the <a href="https://wiki.jenkins.io/display/JENKINS/Support+Core+Plugin" class="external-link" target="_blank" rel="nofollow noopener">support core</a> plugin to provide further information here?</p>

<p>Thanks,<br/>
James</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/vrobert78.jpg" title="vrobert78's avatar" /> vrobert78:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-43106?focusedId=328120&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-328120">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=markewaite" class="user-hover" rel="markewaite">markewaite</a> , since disabling the three plugins (JIRA plugin, JIRA Integration for Blue Ocean and Blue Ocean), hangs have gone. Everything has running fine yesterday.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=markewaite" class="user-hover" rel="markewaite">markewaite</a> , since disabling the three plugins (JIRA plugin, JIRA Integration for Blue Ocean and Blue Ocean), hangs have gone. Everything has running fine yesterday.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/vrobert78.jpg" title="vrobert78's avatar" /> vrobert78:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-43106?focusedId=328122&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-328122">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>We have just completly uninstall plugins (JIRA plugin, JIRA Integration for Blue Ocean and Blue Ocean) and then reinstall them.</p>

<p>Indeed, versions has changed:</p>
<ul>
	<li>Blue Ocean 1.4.1</li>
	<li>JIRA Integration for Blue Ocean 1.4.1</li>
	<li>JIRA Plugin 2.5 (was 2.5.1)</li>
</ul>


<p>We hope this will be as fine as yesterday.</p>
</pre>
</details>

</ul>
</details>

<p>We have just completly uninstall plugins (JIRA plugin, JIRA Integration for Blue Ocean and Blue Ocean) and then reinstall them.</p>

<p>Indeed, versions has changed:</p>
<ul>
	<li>Blue Ocean 1.4.1</li>
	<li>JIRA Integration for Blue Ocean 1.4.1</li>
	<li>JIRA Plugin 2.5 (was 2.5.1)</li>
</ul>


<p>We hope this will be as fine as yesterday.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/markewaite.png" title="markewaite's avatar" /> markewaite:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-43106?focusedId=328159&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-328159">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=vrobert78" class="user-hover" rel="vrobert78">vrobert78</a> thanks very much for your investigation and for sharing your results.  I'd like to close this bug as a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-48357" title="Binary Compatibility between JIRA Plugin and Apache HttpComponents Client 4.x API" class="issue-link" data-issue-key="JENKINS-48357"><del>JENKINS-48357</del></a>.  <a href="https://issues.jenkins.io/browse/JENKINS-48357" title="Binary Compatibility between JIRA Plugin and Apache HttpComponents Client 4.x API" class="issue-link" data-issue-key="JENKINS-48357"><del>JENKINS-48357</del></a> will remain open until the fix is released in the Jira plugin.  Is that OK?</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=vrobert78" class="user-hover" rel="vrobert78">vrobert78</a> thanks very much for your investigation and for sharing your results.  I'd like to close this bug as a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-48357" title="Binary Compatibility between JIRA Plugin and Apache HttpComponents Client 4.x API" class="issue-link" data-issue-key="JENKINS-48357"><del>JENKINS-48357</del></a>.  <a href="https://issues.jenkins.io/browse/JENKINS-48357" title="Binary Compatibility between JIRA Plugin and Apache HttpComponents Client 4.x API" class="issue-link" data-issue-key="JENKINS-48357"><del>JENKINS-48357</del></a> will remain open until the fix is released in the Jira plugin.  Is that OK?</p>

**Comment by jenkins-infra-bot** [comment]
<i>[Original `duplicates` from Jira: <a href="https://github.com/jenkinsci/jenkins/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-48357%22">JENKINS-48357</a>]</i>
